cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/util/ResponseManager.kt

```diff
@@ -14,7 +14,6 @@ internal interface ResponseManager {
     fun parseCustomConsentRes(r: Response): CustomConsentResp
 
     // Optimized
```

Comment:
I suppose this comment was regarding `fun parseMetaDataRes` therefore it can be removed

Reply:
Not really. I think the comment extends to all methods (network calls) bellow it. Either way, the comment is not adding any value, I'll remove it.